apps/desktop/src/lib/config/config.ts

```diff
@@ -60,8 +60,4 @@ export function persistedPRTitle(projectId: string, seriesName: string): Persist
 	return persistWithExpiration('', 'seriesCurrentPRTitle_' + projectId + '_' + seriesName, 5);
 }
```

Comment:
Could we do the same with `persistedPRTitle` and `persistedPRBody`? We are needlessly packaging by layer here rather than feature.

Reply:
Done 👍